src/Accessibility/AXHearingUtilities.cs

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright 2021 Microsoft Corp
+//
+// Authors:
+//   Rachel Kang (rachelkang@microsoft.com)
+//
+
+#nullable enable
+
+#if !TVOS && !MONOMAC
+
+using System;
+using System.Runtime.InteropServices;
+using CoreGraphics;
+using Foundation;
+using ObjCRuntime;
+
+namespace Accessibility {
+    public static partial class AXHearingUtilities {
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        [DllImport (Constants.AccessibilityLibrary)]
+        static extern /* AXHearingDeviceEar */ nint AXMFiHearingDeviceStreamingEar ();
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        [DllImport (Constants.AccessibilityLibrary, EntryPoint = "AXSupportsBidirectionalAXMFiHearingDeviceStreaming")]
+        [return: MarshalAs (UnmanagedType.I1)]
+        static extern bool SupportsBidirectionalStreaming ();
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        [DllImport (Constants.AccessibilityLibrary)]
+        static extern /* NSArray<NSUUID *> * */ IntPtr AXMFiHearingDevicePairedUUIDs ();
+        
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        public static AXHearingDeviceEar GetMFiHearingDeviceStreamingEar ()
+        {
+            return (AXHearingDeviceEar) (long) AXMFiHearingDeviceStreamingEar ();
+        }
+
+        [iOS (15,0), NoMac, NoTV, Watch (8,0)]
+        public static NSUuid[] GetMFiHearingDevicePairedUuids ()
+        {
+            return NSArray.ArrayFromHandle<NSUuid> (AXMFiHearingDevicePairedUUIDs ());
+        }
+    }
```

Comment:
It looks like you've used spaces for indentation - we use tabs.

Reply:
This is still the case :smile:

Reply:
:( so strange... I'll check through everything again!